src/processor/reconnection-graph.service.ts

```diff
@@ -114,31 +127,25 @@ export class ReconnectionGraphService {
         });
 
         if (batch.length > 0) {
-          const payWithCapacityBatchAllOp = ExtrinsicHelper.payWithCapacityBatchAll(providerKeys, batch);
-          promises.push(payWithCapacityBatchAllOp.signAndSend());
+          promises.push(this.blockchainService.createExtrinsic({ pallet: 'frequencyTxPayment', extrinsic: 'payWithCapacityBatchAll' }, {}, providerKeys, batch).signAndSend());
```

Comment:
See other comment about passing event type.

Reply:
good to know